### PR TITLE
update SDK/CLI test for force_rerun

### PR DESCRIPTION
**Pull request by YingChen1996**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

**Comment by YingChen1996** [comment]
need update version in setup.sh

File: sdk/jobs/pipelines/1a_pipeline_with_components_from_yaml/pipeline_with_components_from_yaml.ipynb [hunk 1]
```diff
@@ -209,6 +209,7 @@
     "@dsl.pipeline(\n",
     "    default_compute=\"cpu-cluster\",\n",
     "    description=\"E2E dummy train-score-eval pipeline with components defined via yaml\",\n",
+    "    force_rerun=True,\n",
```
[the remark this answers is not in the document]
Comment (YingChen1996) [reply sdk/jobs/pipelines/1a_pipeline_with_components_from_yaml/pipeline_with_components_from_yaml.ipynb]: reverted and update ci workflow

File: .github/workflows/sdk-jobs-pipelines-2d_image_classification_with_densenet-image_classification_with_densenet.yml [hunk 2]
```diff
@@ -17,30 +17,31 @@ jobs:
...
+        run: bash setup.sh
+        working-directory: cli
+        continue-on-error: true
+      - name: run jobs/pipelines/2d_image_classification_with_densenet/image_classification_with_densenet.ipynb
+        run: |
           mkdir ../../.azureml
           echo '{"subscription_id": "6560575d-fa06-4e7d-95fb-f962e74efd7a", "resource_group": "azureml-examples-rg", "workspace_name": "main"}' > ../../.azureml/config.json 
-          sed -i -e "s/DefaultAzureCredential/AzureCliCredential/g" image_classification_with_densenet.ipynb          
+          sed -i -e "s/DefaultAzureCredential/AzureCliCredential/g" image_classification_with_densenet.ipynb
+          sed -i "s/@dsl.pipeline(/force_rerun=True,&/" image_classification_with_densenet.ipynb
```
Comment (wangchao1230) [inline .github/workflows/sdk-jobs-pipelines-2d_image_classification_with_densenet-image_classification_with_densenet.yml:45]: update the readme.py

File: .github/workflows/sdk-jobs-pipelines-2c_nyc_taxi_data_regression-nyc_taxi_data_regression.yml [hunk 3]
```diff
@@ -17,30 +17,31 @@ jobs:
...
+        run: bash setup.sh
+        working-directory: cli
+        continue-on-error: true
+      - name: run jobs/pipelines/2c_nyc_taxi_data_regression/nyc_taxi_data_regression.ipynb
+        run: |
           mkdir ../../.azureml
           echo '{"subscription_id": "6560575d-fa06-4e7d-95fb-f962e74efd7a", "resource_group": "azureml-examples-rg", "workspace_name": "main"}' > ../../.azureml/config.json 
-          sed -i -e "s/DefaultAzureCredential/AzureCliCredential/g" nyc_taxi_data_regression.ipynb          
+          sed -i -e "s/DefaultAzureCredential/AzureCliCredential/g" nyc_taxi_data_regression.ipynb
+          sed -i "s/@dsl.pipeline(/force_rerun=True,&/" nyc_taxi_data_regression.ipynb
```
Comment (wangchao1230) [inline .github/workflows/sdk-jobs-pipelines-2c_nyc_taxi_data_regression-nyc_taxi_data_regression.yml:45]: This is not working

Comment (YingChen1996) [reply .github/workflows/sdk-jobs-pipelines-2c_nyc_taxi_data_regression-nyc_taxi_data_regression.yml]: modified

File: .github/workflows/sdk-jobs-pipelines-2d_image_classification_with_densenet-image_classification_with_densenet.yml (hunk 2 above)
Comment (YingChen1996) [reply .github/workflows/sdk-jobs-pipelines-2d_image_classification_with_densenet-image_classification_with_densenet.yml]: updated

**Pull request merged**